array.c

```diff
@@ -1232,19 +1277,21 @@ rb_ary_cat(VALUE ary, const VALUE *argv, long len)
 
 /*
  *  call-seq:
- *     ary.push(obj, ...)    -> ary
- *     ary.append(obj, ...)  -> ary
+ *    ary.push(*objects)    -> self
+ *    ary.append(*objects)  -> self
  *
- *  Append --- Pushes the given object(s) on to the end of this array. This
- *  expression returns the array itself, so several appends
- *  may be chained together. See also Array#pop for the opposite
- *  effect.
+ *  Appends each argument in <tt>*objects</tt> to <tt>ary</tt>;  returns <tt>self</tt>:
```

Comment:
The `*` has no meaning inside the method, it's an Array so this should be "Appends each argument in +objects+ to +ary+"

Reply:
But *objects in the call-seq is still ok?

Reply:
Yes, so the user knows they can pass in an arbitrary number of things and they'll get collected together.

Reply:
Done.